shortfin/dev_me.py

```diff
@@ -196,7 +197,8 @@ def configure_mode(env_info: EnvInfo, args):
         env_vars["SHORTFIN_IREE_SOURCE_DIR"] = env_info.iree_dir
     if env_info.clang_exe:
         env_vars["CC"] = env_info.clang_exe
-        env_vars["CXX"] = f"{env_info.clang_exe}++"
+        clangxx_exe = "clang++-".join(env_info.clang_exe.split("clang-"))
```

Comment:
Does this work for unadorned "clang" commands? Maybe drop this file from the patch and we do it separately?